service/pom.xml

```diff
@@ -134,6 +134,12 @@
             <version>1.9.5</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.opentestsystem.common</groupId>
+            <artifactId>tds-common-legacy</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
```

Comment:
Can we use the `${tds.common}` property?  legacy and common should never be out of sync version wise.